src/main/java/frc/robot/sensors/AnalogSelector.java

```diff
@@ -0,0 +1,120 @@
+package frc.robot.sensors;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.Sendable;
+import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
+import edu.wpi.first.wpilibj.smartdashboard.SendableRegistry;
+
+public class AnalogSelector implements Sendable {
+
+    private static final double MODE_ONE_LOW_VOLTAGE = 2.5;
+    private static final double MODE_ONE_HIGH_VOLTAGE = 2.75;
+
+    private static final double MODE_TWO_LOW_VOLTAGE = 2.75;
+    private static final double MODE_TWO_HIGH_VOLTAGE = 3.1;
+
+    private static final double MODE_THREE_LOW_VOLTAGE = 3.1;
+    private static final double MODE_THREE_HIGH_VOLTAGE = 3.5;
+
+    private static final double MODE_FOUR_LOW_VOLTAGE = 3.5;
+    private static final double MODE_FOUR_HIGH_VOLTAGE = 3.75;
+
+    private static final double MODE_FIVE_LOW_VOLTAGE = 3.75;
+    private static final double MODE_FIVE_HIGH_VOLTAGE = 3.95;
+
+    private static final double MODE_SIX_LOW_VOLTAGE = 3.95;
+    private static final double MODE_SIX_HIGH_VOLTAGE = 4.1;
+
+    private static final double MODE_SEVEN_LOW_VOLTAGE = 4.1;
+    private static final double MODE_SEVEN_HIGH_VOLTAGE = 4.2;
+
+    private static final double MODE_EIGHT_LOW_VOLTAGE = 4.2;
+    private static final double MODE_EIGHT_HIGH_VOLTAGE = 4.3;
+    
+    private static final double MODE_NINE_LOW_VOLTAGE = 4.3;
+    private static final double MODE_NINE_HIGH_VOLTAGE = 4.4;
+
+    private static final double MODE_TEN_LOW_VOLTAGE = 4.4;
+    private static final double MODE_TEN_HIGH_VOLTAGE = 4.5;
+
+    private static final double MODE_ELEVEN_LOW_VOLTAGE = 4.5;
+    private static final double MODE_ELEVEN_HIGH_VOLTAGE = 4.6;
+
+    private static final double MODE_TWELVE_LOW_VOLTAGE = 4.6;
+    private static final double MODE_TWELVE_HIGH_VOLTAGE = 5;
+
+    private static final Range[] VOLTAGE_RANGES = {
+        new Range(0, 2.5),
+        new Range(MODE_ONE_LOW_VOLTAGE, MODE_ONE_HIGH_VOLTAGE), 
+        new Range(MODE_TWO_LOW_VOLTAGE, MODE_TWO_HIGH_VOLTAGE),
+        new Range(MODE_THREE_LOW_VOLTAGE, MODE_THREE_HIGH_VOLTAGE), 
+        new Range(MODE_FOUR_LOW_VOLTAGE, MODE_FOUR_HIGH_VOLTAGE), 
+        new Range(MODE_FIVE_LOW_VOLTAGE, MODE_FIVE_HIGH_VOLTAGE),
+        new Range(MODE_SIX_LOW_VOLTAGE, MODE_SIX_HIGH_VOLTAGE), 
+        new Range(MODE_SEVEN_LOW_VOLTAGE, MODE_SEVEN_HIGH_VOLTAGE), 
+        new Range(MODE_EIGHT_LOW_VOLTAGE, MODE_EIGHT_HIGH_VOLTAGE),
+        new Range(MODE_NINE_LOW_VOLTAGE, MODE_NINE_HIGH_VOLTAGE), 
+        new Range(MODE_TEN_LOW_VOLTAGE, MODE_TEN_HIGH_VOLTAGE), 
+        new Range(MODE_ELEVEN_LOW_VOLTAGE, MODE_ELEVEN_HIGH_VOLTAGE),
+        new Range(MODE_TWELVE_LOW_VOLTAGE, MODE_TWELVE_HIGH_VOLTAGE), 
+        //new Range(4.6, 5)
```

Comment:
Does this provide future value? If so it would be worth adding a comment, otherwise I would 🔥

Reply:
I was using this as a reference for what to set as the last few (MODE_TWELVE) constants and left it there by accident. Thanks for catching that.